lib/galaxy/config/sample/datatypes_conf.xml.sample

```diff
@@ -142,8 +142,12 @@
     <datatype extension="d3_hierarchy" type="galaxy.datatypes.text:Json" mimetype="application/json" subclass="true" display_in_upload="true"/>
     <datatype extension="cytoscapejson" type="galaxy.datatypes.text:CytoscapeJson" mimetype="application/json" display_in_upload="true" description="Cytoscape JSON format for network visualization, typically containing 'nodes' and 'edges' in a JSON object." description_url="https://js.cytoscape.org/#notation/elements-json" />
     <datatype extension="imgt.json" type="galaxy.datatypes.text:ImgtJson" mimetype="application/json" display_in_upload="True"/>
-    <datatype extension="geojson" type="galaxy.datatypes.text:GeoJson" mimetype="application/json" display_in_upload="True"/>
-    <datatype extension="vitesscejson" type="galaxy.datatypes.text:VitessceJson" mimetype="application/json" display_in_upload="True"/>
+    <datatype extension="geojson" type="galaxy.datatypes.text:GeoJson" mimetype="application/json" display_in_upload="True">
+      <visualization plugin="openlayers" />
```

Comment:
This should be Kepler.gl. Is there a reason we don't just add this as datatype attribute: `visualization="kepler"`?

Reply:
Discussed this in the team meeting and decided to keep it as is, as separate tag.

Reply:
Also recommending to keep using `openlayers` as default view for geojson since it loads faster than Kepler. Thx again for all these additions.